db/migrate/20240228032657_add_hook_order_to_plugin.rb

```diff
@@ -0,0 +1,5 @@
+class AddHookOrderToPlugin < ActiveRecord::Migration[5.2]
+  def change
+    add_column :plugins, :hook_priority, :integer, :default => 10
```

Comment:
I think it's more natural to use `0` as the default value, but 10 is also good.
For example, comparing the following to cases, "b" looks straightforward to me. 
a.`9` is earlier than the default and `11` is later than the default.
b. `-1` is earlier than the default and `1` is later than the default.